ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -89,3 +89,52 @@
         chdir: "{{ cifmw_artifacts_basedir }}/logs/edpm"
       changed_when: true
       ignore_errors: true
+
+    - name: Create a file to store the operator images
+      changed_when: true
+      ignore_errors: true
+      ansible.builtin.shell: |
```

Comment:
we can use ansible.builtin.copy, if really:
```YAML
ansible.builtin.copy:
  dest: "{{ cifmw_artifacts_basedir }}/artifacts/operator_images.yaml"
  content: "cifmw_install_yamls_vars:"
```

Reply:
updated